### ISSUE TITLE
Does not work Windows Anki Version ⁨2.1.49 (dc80804a)⁩

### ISSUE THREAD

**Issue by jackfang888** [matthayes/anki_flex_dupes#3]
<br>

### Images will probably describe the problem better than I can with words:

<br>

<br>

![image](https://user-images.githubusercontent.com/20893390/150426786-0fc70ff0-17bc-40da-b832-61a9de579c5c.png)

Though I have added the `_pk` string to the end of the `Text` field name, it still flags as a duplicate.  However if I move the `Text_pk` field to the first field in the Note Type, it is no longer flagged as a duplicate:

![image](https://user-images.githubusercontent.com/20893390/150428244-de2120fb-5941-4bc4-963d-883e9a8d0768.png)

Since Anki does not flag duplicates when the Text field is the first field, then it must consider that field to be unique.  Therefore adding `_pk` to it and moving something else to the first field should still result in no duplicates, however that is not the case.

Furthermore, if I add `_pk` to the `Source` field (which should force Anki to check that field as well) it does not flag it even though it *is* a duplicate:

![image](https://user-images.githubusercontent.com/20893390/150432522-fc8f6850-3783-4cba-bc38-40fd8bf6111c.png)

**Comment by kanjieater** [comment]
Ran into this too. The wrapper parameters changed:

```
def setup():
    Editor.checkValid = wrap(Editor.checkValid, checkValid, "after")
    Note.fields_check = wrap(Note.fields_check, dupeOrEmpty, "after")
    Editor.showDupes = showDupes
    Editor._links["dupes"] = showDupes
```

Also remove anywhere you see "_old" in a function signature, and it should go back to working (at least it does for me though my code is highly modified).

**Comment by jackfang888** [comment]
Thank you @kanjieater but I know almost nothing about programming.  Hopefully @matthayes will see this and make an update.

**Comment by BrucePlumb** [comment]
Yeah! it works now! Thank you kanjieater. Corrected the first two lines of the def setup(): and there were just two function signatures with "_old" that I removed (including the preceding comma). I know nothing about python but this is something anybody could do.

**Comment by brettz9** [comment]
@kanjieater : If this plugin is no longer maintained, might you consider publishing your fork?

**Comment by kanjieater** [comment]
It's been a while but I think I took things into a more customized direction. I can look at some point into publishing my variant though
